_pages/en_US/include/seedminer-base.txt

```diff
@@ -67,6 +67,7 @@ In this section, you will use the Bruteforce Movable website to retrieve your de
   + This is usually fast (1-5 minutes)
   + During this process, the bot you added may not show up on your 3DS. As long as the website updates, this is not an issue
   + If you are still waiting after half an hour, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask (in English) for someone there to assist you
+  + If you get "We were unable to successfully complete your bruteforce request. :`(" and you have a New 3DS / New 3DS XL / New 2DS XL (as indicated by the four shoulder buttons on the back and the C-Stick on the right), you can follow [Homebrew Launcher (super-skaterhax)](homebrew-launcher-(super-skaterhax)) instead of this method
```

Comment:
imo we should link to troubleshooting here, we have a huge detailed section on what happens with it.

Reply:
i would also rather link to troubleshooting here where we could potentially add an image of the n3ds series just to prevent any confusion